tests/test_app/tests/upload_tests.py

```diff
@@ -82,7 +82,6 @@ def test_metadata(self):
         metadata_attrs = {
             "Accept-Ranges",
             "Content-Length",
-            "Content-Security-Policy",
```

Comment:
This test was failing when run against a new version of Minio server (it was passing on the 4-year-old Minio server used in CI).

Apparently Minio server is no longer returning [a `Content-Security-Policy` header by default](https://min.io/docs/minio/linux/reference/minio-server/settings/console.html#envvar.MINIO_BROWSER_CONTENT_SECURITY_POLICY). I don't think this is a problem, since this behavior is outside the scope of what this library needs to care about.

@thomasf Let me know if I'm wrong.